Sources/Services/AWSS3/Sources/AWSS3/S3Client+S3Express.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// Code generated by smithy-swift-codegen. DO NOT EDIT!
+
+import protocol AWSSDKIdentity.S3ExpressCreateSessionClient
+import protocol ClientRuntime.DefaultClientConfiguration
+import struct AWSSDKIdentity.S3ExpressIdentity
+
+
```

Comment:
This is a code-generated, concrete implementation of the `AWSSDKIdentity.S3ExpressCreateSessionClient` protocol.

This serves as a "dependency inversion" wrapper to allow `AWSSDKIdentity` to call the S3 client without depending on it.

The method below takes a `DefaultClientConfiguration` instead of the concrete S3 configuration type, then it casts it to S3 config.  Because this wrapper is only ever generated for S3, the cast to `S3Client.Config` should never fail.